python/arcticdb/version_store/_store.py

```diff
@@ -556,7 +559,7 @@ def write(
                 return None
             else:
                 vit = self.version_store.write_versioned_dataframe(
-                    symbol, item, norm_meta, udm, prune_previous_version, sparsify_floats
+                    symbol, item, norm_meta, udm, prune_previous_version, sparsify_floats, ensure_sorted
```

Comment:
Consider creating an options object that passes from Python all the way to the `*_impl` in C++.